runtime/near-vm-runner-standalone/src/main.rs

```diff
@@ -163,28 +164,26 @@ fn main() {
     step.promise_results(promise_results);
 
     let mut results = script.run();
-    let (outcome, err) = results.outcomes.pop().unwrap();
-
-    println!(
-        "{:#?}",
-        StandaloneOutput {
-            outcome: outcome.clone(),
-            err: err.map(|it| it.to_string()),
-            state: State(results.state.fake_trie),
-        }
-    );
-
-    if let Some(outcome) = &outcome {
-        println!("\nLogs:");
-        for log in &outcome.logs {
-            println!("{}\n", log);
-        }
-    }
-
-    match &outcome {
-        Some(outcome) => {
+    let last_result = results.outcomes.pop().unwrap();
+    let maybe_error = last_result.error();
+
+    match &last_result {
```

Comment:
Minor minor nit, but since `last_result` is no longer used after this, you can match by value here and avoid the `clone` of the `VMOutcome` in the `Aborted | Ok` branch.

Reply:
Yeah that clone annoys me, too. The problem is that `output` is used again for printing afterwards. So it is not possible to move it when constructing `StandaloneOutput`, without either reordering the prints or changing the `StandaloneOutput::output` type to a borrow. Neither of which seems to be worth it just to avoid the clone here.